Sources/Views/EpisodeVideoView.swift

```diff
@@ -66,6 +66,13 @@ public func videoView(
             player.setCurrentTime(time)
             if (play) { player.play() }
           }
+        
+          player.setPlaybackRate( localStorage.getItem('pf_playbackrate') || 1.0
+          );
+          player.on('playbackratechange', function(data) {
+            localStorage.setItem('pf_playbackrate', data.playbackRate);
+          });
```

Comment:
I'm not an expert on JS, but this code looks solid to me. To make it even more bulletproof, we could ensure type safety by converting the value from `localStorage` to a number. Also, adding error handling when saving data to `localStorage` could be good, for example to handle a case when a user has disabled it.

```suggestion
          const defaultPlaybackRate = 1.0;
          const playbackRate = parseFloat(localStorage.getItem('pf_playbackrate')) || defaultPlaybackRate;
          player.setPlaybackRate(playbackRate);

          player.on('playbackratechange', function(data) {
            try {
              localStorage.setItem('pf_playbackrate', String(data.playbackRate));
            } catch (e) {
              console.error('Failed to save playback rate:', e);
            }
          });
```